hydroflow/src/lib.rs

```diff
@@ -46,4 +46,6 @@ mod booktest {
     booktest!(surface_flows);
```

Comment:
what's with the empty chat_driver.py file above?

Reply:
`rwx` permissions changed only, will try to remove the change